pyproject.toml

```diff
@@ -238,3 +238,9 @@ enable_error_code = ['explicit-override']
 ignore_missing_imports = true
 warn_redundant_casts = true
 warn_unused_ignores = true
+
+[tool.codespell]
```

Comment:
I first tried with a bigger context, and that was annoying, but:
```
after-context = 1
before-context = 1
```
at least gives me an idea if the error is in the middle of a code block or a doc string.

Reply:
Err, uhm, hmm. I tried it, and I don't love the output it gives. Some extra context can be helpful, but so many of them were blank lines, and it makes it much harder to scan down the list.